hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -3498,13 +3511,13 @@ public SplitOrMergeTracker getSplitOrMergeTracker() {
   }
 
   @Override
-  public LoadBalancer getLoadBalancer() {
+  public RSGroupBasedLoadBalancer getLoadBalancer() {
```

Comment:
Yeah, can't just return 'LoadBalancer' only now LB handles RSGroups too? (Maybe not possible for RSGroupLB...)